presqt/api_v1/utilities/validation/token_validation.py

```diff
@@ -20,6 +20,27 @@ def token_validation(request):
     try:
         return request.META['HTTP_PRESQT_SOURCE_TOKEN']
     except KeyError:
-        raise PresQTAuthorizationError(
+        raise PresQTValidationError(
             "'presqt-source-token' missing in the request headers.",
+            status.HTTP_400_BAD_REQUEST)
+
+def destination_token_validation(request):
```

Comment:
I'm not sure if `validation` is the best wording here.
`destination_token_validation` (similar to the previous function) to me implies that something is being done to verify the token - not (just) seeing that it exists.

Perhaps a different name on these functions would make it more clear.